src/guides/v2.4/extension-dev-guide/message-queues/config-mq.md

```diff
@@ -116,6 +116,9 @@ The [`consumers-wait-for-messages`]({{page.baseurl}}/install-gde/install/cli/ins
 The problem is that every time the cron job `cron_consumers_runner` runs, it spawns a new consumer process, the consumer checks if messages are available, and it terminates itself if there are no messages.
 Meanwhile, the `onlySpawnWhenMessageAvailable` attribute first checks if there are available messages, and it spawns a new consumer process only if there are messages. It means that it does not spawn unneeded processes which take up memory, live for a very short period, and then disappear.
 
+{:.bs-callout-warning}
+The [`consumers-wait-for-messages`]({{page.baseurl}}/install-gde/install/cli/install-cli-subcommands-consumers.html) option is a global option and cannot be configured separately for each consumer, such as the `onlySpawnWhenMessageAvailable` option.
```

Comment:
The "Configure consumer behavior" topic linked above should also be updated to note that the --consumers-wait-for-messages option is global.

Reply:
Good call. Made it